lib/slack.ex

```diff
@@ -157,9 +165,94 @@ defmodule Slack do
     end
   end
 
+  @doc ~S"""
+  Turns a string like `"@USER_NAME"` into the ID that Slack understands (`"U…"`).
+  """
+  def lookup_user_id("@" <> user_name, slack) do
```

Comment:
I also agree that it should be converted to a Struct. I'll open an issue for that.

> Should the send_*() function go into their own module? Would it be the same module as the lookups (which they now use) or a different one?

I think it'd be nice if they were in their own modules. I assume we'll want to `import` at least the `send_*` functions so we can use them without having to `alias` or call them with their module each time.